addons/crm_addons/models/crm_lead.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+class Lead(models.Model):
+    _inherit = 'crm.lead'
+
+    @api.model
+    def form_view_auto_fill(self):
```

Comment:
If it's not removed, it's got no reason to be callable over rpc so should have an `_` prefix.